<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TransactionsPage</class>
 <widget class="QWidget" name="TransactionsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>902</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#TransactionsPage {
	background-color:#f3f4f7;
}

#filterWidget {
	background-color: white;
}

QLineEdit {
	border: none;
	border-bottom: 2px solid #3cb0e5;
	padding: 3px 5px;
	background: transparent;
}

QComboBox::drop-down {
    subcontrol-origin: margin;
    subcontrol-position: right center;
    width: 15px;
    height: 15px;
    border: none;
}

QComboBox {
	border: none;
	border-bottom: 2px solid #3cb0e5;
	padding: 3px 5px;
	background: white;
	outline: none;
}

QComboBox:editable {
	background: white;
}

QComboBox:on {
	padding-top: 3px;
	padding-left: 4px;
	padding-right: 15px;
}

QComboBox::down-arrow {
	image: url(:/icons/md/arrow_down);
	width: 20px;
	height: 20px;
}

QComboBox::down-arrow:on {
	image: url(:/icons/md/arrow_up);
	width: 20px;
	hieght: 20px;
}

#bodyWidget, #titleWidget, #dateRangeWidget {
	background-color: white;
}

QTableView::item {
	border: 0;
	padding: 5px 15px;
}

QTableView::item:selected {
	background-color: #3cb0e5;
}

QHeaderView::section {
	font-family: 'Montserrat';
	font-size: 10px;
	text-transform: uppercase;
	padding: 5px 20px;
	border: none;
}

QHeaderView::down_arrow {
	image: url(:/icons/md/arrow_down);
}

QHeaderView::up_arrow {
	image: url(:/icons/md/arrow_up);
}

QDateEdit {
    border: none;
	height: 24px;
    border-bottom: 2px solid #3cb0e5;
    padding: 3px 5px;
    background: white;
    outline: none;
    padding-right: 20px;
}

QDateEdit::drop-down {
    image: url(:/icons/md/arrow_down);
	subcontrol-origin: margin;
	subcontrol-position: right center;
	width: 20px;
	height: 20px;
	border: none;
    padding-bottom: 2px;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="spacing">
    <number>1</number>
   </property>
   <property name="leftMargin">
    <number>20</number>
   </property>
   <property name="topMargin">
    <number>20</number>
   </property>
   <property name="rightMargin">
    <number>20</number>
   </property>
   <property name="bottomMargin">
    <number>20</number>
   </property>
   <item>
    <widget class="QWidget" name="titleWidget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>60</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="titleWidgetLayout">
      <property name="leftMargin">
       <number>20</number>
      </property>
      <property name="topMargin">
       <number>20</number>
      </property>
      <property name="rightMargin">
       <number>20</number>
      </property>
      <property name="bottomMargin">
       <number>20</number>
      </property>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <family>Montserrat</family>
          <pointsize>14</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>TRANSACTION HISTORY</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="../bitcoin.qrc">:/icons/md/question_mark_blue</pixmap>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="bodyWidget" native="true">
     <layout class="QVBoxLayout" name="bodyWidgetLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="filterWidget" native="true">
        <layout class="QHBoxLayout" name="filterWidgetLayout" stretch="1,1,5,2">
         <property name="spacing">
          <number>40</number>
         </property>
         <property name="leftMargin">
          <number>74</number>
         </property>
         <property name="topMargin">
          <number>20</number>
         </property>
         <property name="rightMargin">
          <number>20</number>
         </property>
         <property name="bottomMargin">
          <number>20</number>
         </property>
         <item>
          <widget class="QComboBox" name="dateWidget">
           <property name="minimumSize">
            <size>
             <width>115</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>115</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Open Sans</family>
             <pointsize>13</pointsize>
            </font>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="typeWidget">
           <property name="minimumSize">
            <size>
             <width>115</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>115</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Open Sans</family>
             <pointsize>13</pointsize>
            </font>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="addressWidget">
           <property name="font">
            <font>
             <family>Open Sans</family>
             <pointsize>13</pointsize>
            </font>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="amountWidget">
           <property name="minimumSize">
            <size>
             <width>180</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>180</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Open Sans</family>
             <pointsize>13</pointsize>
            </font>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QTableView" name="transactionView">
        <property name="font">
         <font>
          <family>Open Sans</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="contextMenuPolicy">
         <enum>Qt::CustomContextMenu</enum>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="lineWidth">
         <number>0</number>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOn</enum>
        </property>
        <property name="tabKeyNavigation">
         <bool>false</bool>
        </property>
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="showGrid">
         <bool>false</bool>
        </property>
        <attribute name="horizontalHeaderMinimumSectionSize">
         <number>25</number>
        </attribute>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="dateRangeWidget" native="true">
     <layout class="QHBoxLayout" name="rangeWidgetLayout">
      <property name="leftMargin">
       <number>20</number>
      </property>
      <property name="topMargin">
       <number>20</number>
      </property>
      <property name="rightMargin">
       <number>20</number>
      </property>
      <property name="bottomMargin">
       <number>20</number>
      </property>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <family>Montserrat</family>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>RANGE FROM</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="dateFrom">
        <property name="font">
         <font>
          <family>Open Sans</family>
          <pointsize>13</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <family>Montserrat</family>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>TO</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="dateTo">
        <property name="font">
         <font>
          <family>Open Sans</family>
          <pointsize>13</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../bitcoin.qrc"/>
 </resources>
 <connections/>
</ui>
